pom.xml

```diff
@@ -109,22 +114,60 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-instrumentation-annotations</artifactId>
+            <version>2.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-sdk-trace</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-sdk-metrics</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-exporter-otlp</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentelemetry</groupId>
+            <artifactId>opentelemetry-exporter-logging</artifactId>
+        </dependency>
```

Comment:
these imports should all be replaced by a dependency on the opentelemetry plugin so that multiple versions aren't bundled into Jenkins causing problems:
https://plugins.jenkins.io/opentelemetry/dependencies/

Reply:
I tried to do as you suggested, but that plugin is not available at
* https://repo.jenkins-ci.org/public/org/jenkins-ci/plugins/

Do you know the artifact details for the maven pom.xml?

Reply:
Its on the page I linked

Reply:
Thanks, I was able to make it work, but I had to replace all the @WithSpan annotations that I had in the Java code and use manual tracing instead 🤷‍♂️